lib/import-export-visitor.js

```diff
@@ -598,6 +613,25 @@ function hoistExports(visitor, exportDeclPath, mapOrString, childName) {
   visitor.madeChanges = true;
 }
 
+function isConstExportDeclaration(exportDecl) {
```

Comment:
This is really about live binding not const though right? The export default declaration isn't a const so the name is a bit confusing. Maybe `isLiveBound`?